Python news reader

KmrAnish04 opened the pull request:
# Description

A python script which will fetch news from newsapi.com and will speak that news with headlines and description.

Fixes #475


## Type of change

_Please delete options that are not relevant._

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Documentation Update

# Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

KmrAnish04 commented:
HI @HarshCasper , Thanks for assigning this issue to me and here is my PR for that issue. There is showing "All checks have failed", what does it mean? or I have made this PR in wrong way? . Let me know everything has done right or not. If all right, please merge this small contribution from my side. Thankful to you @HarshCasper .

KmrAnish04 commented:
Hi @seema1711 , Can look into this PR, Why this is showing "All checks have failed" ? 
Is something wrong in my contribution? . Please, let me know with solution.

KmrAnish04 commented:
Hi @HarshCasper , Please look into this PR's and let me if something is missing. If all right, then merge this small contribution from my side. Thanks.

HarshCasper commented on Python/Python_news_reader/Python news reader.py:
```diff
@@ -0,0 +1,44 @@
+                    speaker(description)
+
+if __name__ == "__main__":
+    func()
```
Kindly add a new line signifying the End of File

HarshCasper commented on Python/Python_news_reader/Python news reader.py:
```diff
@@ -0,0 +1,44 @@
+
+def func():
+    url = (
+        'http://newsapi.org/v2/top-headlines?country=in&category=business&apiKey=paste your API key here')  # Only paste your API key here, don't change the whole link.
```
Put the Comment above the Line of Code to improve readibility

HarshCasper commented on Python/Python_news_reader/Python news reader.py:
```diff
@@ -0,0 +1,44 @@
+                    time.sleep(1)
+                    speaker(description)
+
+if __name__ == "__main__":
```
Add a `requirements.txt` for your Script

HarshCasper commented on Python/Python_news_reader/Readme.md:
```diff
@@ -0,0 +1,11 @@
+### -What is News API?
+
+News API is a simple HTTP REST API for searching and retrieving live articles from all over the web.
+Visit the newsapi.org for documentaion and your personal API key.
```
Add a brief note on how someone can run your Script for better accessibility?

HarshCasper commented on Python/Python_news_reader/Python news reader.py:
```diff
@@ -0,0 +1,44 @@
+    response = requests.get(url)
+    m = response.json()
+    print(m)
+    lst = m['articles']
```
The Variable names don't seem to follow any specification. Try running your Script through PyLint and make apt changes.

HarshCasper reviewed (changes requested):
Kindly make the suggested changes

KmrAnish04 commented:
Hello @HarshCasper , Thanks for your response. I will definitely improve my files according to your expectations and to increase the efficiency and readability of code and documentations.  I am a newbie so just give me some time. Thanks for your cooperation.

KmrAnish04 commented:
Hi @HarshCasper , I closed this PR by mistake, I am sorry about that. And the changes you asked to do in the above files is almost done by me. So now should I create a new pull request for these above changes that you have asked or there another way of doing it. Let me know how can  and work is almost done according to your expectations.

KmrAnish04 commented:
> ## I have modified all the files and added some new files according to expectations and to improve the efficiency of the program.

### Changes done by me:

* Added a new line signifying the End of File
* Added a requirements.md file for this Script
* Added a brief note on how someone can run this Script
* Improved variable names for better understanding
* Added "Url.txt" file for better efficiency
* Improved readme.md file

#### Link to new PR #511 

@HarshCasper
